gpkit/constraints/linked.py

```diff
@@ -1,87 +0,0 @@
-"Implements LinkedConstraintSet"
-from .set import ConstraintSet
-from ..varkey import VarKey
-from .. import SignomialsEnabled
-
-
-class LinkedConstraintSet(ConstraintSet):
```

Comment:
totally support this removal. Note this will break some untested models. See e.g. https://github.com/hoburg/jho/issues/8.